src/components/book.css

```diff
@@ -0,0 +1,170 @@
+.book-panel {
```

Comment:
[OPTIONAL] I think the gray background on the template it has the purpose of emphasizing the design and giving it a modern look, but I do not think the author of the design thought of that background as a feature of the design. Of course, I cannot be certain about it. What do you think? Is it intended? If it is not, maybe you would like to remove it.

[I will leave you a link to mine to give you a reference](https://cranky-jepsen-7a2b4a.netlify.app/), but I do invite you to leave it the way you did it if that's your desire.

Reply:
Yes, you are right.
Yet this one is different from what all Microverse students have.

Thanks a lot, Daniel!